### DISCUSSION TITLE
moxie i have something to say

### DISCUSSION THREAD

**Discussion by daveandbambifan1234** [JordanSantiagoYT/FNF-JS-Engine#620] [General]
I'm stupid and there's nothing we can do about it ):
![image](https://github.com/user-attachments/assets/f47455be-914b-409e-bb40-abab3061f7f2)

I hope this message was useful to you moxie, and have a nice day

**Comment by daveandbambifan1234** [comment]
I apologize to you @JordanSantiagoYT for me being braindead

Moxie got tired of me so it's best to just make to show bugs more detailed as possible and reasonable bugs

**Comment by KernelTech11** [comment]
@moxie-coder see this

**Reply by moxie-coder** [reply]
Ok

**Comment by moxie-coder** [comment]
> I'm stupid and there's nothing we can do about it ): ![image](https://private-user-images.githubusercontent.com/159091869/371399257-f47455be-914b-409e-bb40-abab3061f7f2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0Njg5MzYsIm5iZiI6MTcyNzQ2ODYzNiwicGF0aCI6Ii8xNTkwOTE4NjkvMzcxMzk5MjU3LWY0NzQ1NWJlLTkxNGItNDA5ZS1iYjQwLWFiYWIzMDYxZjdmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyN1QyMDIzNTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMmVkYWZiOGFkZjE5YWY4ZmI0OGYxMjc5YzlkNjFiMGQ5NWMzZjUxMjc0ZTk4Y2RjZDM5NWI1MjJmYzQ3ZWQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5w7qzDGMLyrd4E0AUl4YqTmkRHw9o3UAHX365eGflB8)
> 
> I hope this message was useful to you moxie, and have a nice day

![IMG_4139](https://github.com/user-attachments/assets/d9d40888-e1b1-4a56-9fcd-8527002e41c5)
